docs/pages/ai-assist.mdx

```diff
@@ -83,20 +83,20 @@ Do not share your secret API keys in publicly accessible areas.
 ## Step 2/3. Enable Assist
 
 To enable Teleport Assist, you need to provide your OpenAI API key. On each
-Proxy Service host, perform the following actions:
+Auth Service host, perform the following actions:
```

Comment:
This is not exactly true. Right now OpenAI key is required in Auth and Proxy.

Reply:
so you need it in both `auth_service` and `proxy_service`?

Reply:
Yes, added to this PR.